third_party/ObjexxFCL/src/ObjexxFCL/numeric.hh

```diff
@@ -86,140 +52,6 @@ TINY( T const & )
 	return std::numeric_limits< T >::min();
 }
 
-template< typename T >
-inline
-T
-EPSILON( T const & )
-{
-	static_assert( std::is_floating_point< T >::value, "Floating point argument required" );
-	return std::numeric_limits< T >::epsilon();
-}
-
-template< typename T >
-inline
-int
-PRECISION( T const & x )
-{
-	static_assert( std::is_floating_point< T >::value, "Floating point argument required" );
-	return int( ( DIGITS( x ) - 1 ) * std::log10( double( RADIX( x ) ) ) ); // + 1 if radix(x) == 10^N but we don't support non-traditional architectures
-}
-
-template< typename T >
-inline
-int
-PRECISION( std::complex< T > const & x )
-{
-	static_assert( std::is_floating_point< T >::value, "Floating point argument required" );
-	return int( ( DIGITS( x.real() ) - 1 ) * std::log10( double( RADIX( x.real() ) ) ) ); // + 1 if radix(x) == 10^N but we don't support non-traditional architectures
-}
-
-inline
-int
-EXPONENT_RANGE( std::int8_t const & )
-{
-	return 2;
-}
-
-inline
-int
-EXPONENT_RANGE( std::int16_t const & )
-{
-	return 4;
-}
-
-inline
-int
-EXPONENT_RANGE( std::int32_t const & )
-{
-	return 9;
-}
-
-inline
-int
-EXPONENT_RANGE( std::int64_t const & )
-{
-	return 18;
-}
-
-inline
-int
-EXPONENT_RANGE( std::uint8_t const & )
-{
-	return 2;
-}
-
-inline
-int
-EXPONENT_RANGE( std::uint16_t const & )
-{
-	return 4;
-}
-
-inline
-int
-EXPONENT_RANGE( std::uint32_t const & )
-{
-	return 9;
-}
-
-inline
-int
-EXPONENT_RANGE( std::uint64_t const & )
-{
-	return 18;
-}
-
-inline
-int
-EXPONENT_RANGE( float const & )
-{
-	return 37;
-}
-
-inline
-int
-EXPONENT_RANGE( double const & )
-{
-	return 307;
-}
-
-inline
-int
-EXPONENT_RANGE( long double const & )
-{
-	return 4931; // Assumes long double standing in for REAL(16)
-}
-
-inline
-int
-EXPONENT_RANGE( std::complex< float > const & )
-{
-	return 37;
-}
-
-inline
-int
-EXPONENT_RANGE( std::complex< double > const & )
-{
-	return 307;
-}
-
-inline
-int
-EXPONENT_RANGE( std::complex< long double > const & )
-{
-	return 4931;
```

Comment:
I think EnergyPlus should move to `Real128`. It's more precise. BTW 10E4931 is just a tad (10E4850) larger than the number of atoms in the universe. I wonder what this type of exponent is actually needed for.